### PR TITLE
Fix for deprecated ListItem title

### PR DESCRIPTION
**Pull request by marklchaves**
Fix for deprecated ListItem title in the RN tutorial.

Reference: https://reactnativeelements.com/blog/2020/08/10/2.3-upgrade-guide/#listitem

## Pull Request Info

Running the RN tutorial throws the following warning.

`[Sat Jun 19 2021 05:03:22.951]  WARN     'ListItem.title' prop has been deprecated and will be removed in the next version. `

This PR updates the `ListItem` code according to the following React Native Elements upgrade guide.

https://reactnativeelements.com/blog/2020/08/10/2.3-upgrade-guide/#listitem

### Jira

N/A

### Staged Changes (Requires MongoDB Corp SSO)

N/A

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

**Comment by nathan-contino-mongo** [comment]
@mongomoe @nlarew can one of you take a look at this? Unfortunately my RN skills aren't quite up to snuff for a change of this size.

**Comment by nathan-contino-mongo** [comment]
@marklchaves can you agree to our CLA? https://docs.google.com/forms/d/e/1FAIpQLSeQ9ROFaTu9pyrmPhXc-dEnLD84DbLuT_-tPNZDOL9J10tOKQ/viewform

**Comment by marklchaves** [comment]
Hello @nathan-contino-mongo,

I've signed the CLA.

Thanks so much :-)

**Comment by mongomoe** [comment]
@marklchaves Thanks for contributing with this pull request. This PR uses an updated version of 'react-native-elements' but you didn't push any changes to the package.json. Can you add those changes?

Also, it looks like some of the styling on the list items is not matching the original styling. For instance, when you click a task in your branch the actions spacing and font do not match the original.

Original:
Screen Shot 2021-07-02 at 3 24 46 PM

marklchaves:listitem-ttile-attribute-fix:
Screen Shot 2021-07-02 at 3 19 03 PM


In addition, it appears as if the ListItem is not showing when items are marked "In Progress" or "Complete" in your version.

So required fixes before giving this an approval is:
- UI fixes
- package.json change to upgrade 'react-native-elements'

File: tutorial/rn/components/TaskItem.js [hunk 1]
```diff
@@ -63,21 +63,26 @@ export function TaskItem({ task }) {
...
-      />
+      <ListItem>
+        <ListItem.Content>
+          <ListItem.Title
+            key={task.id}
+            onPress={() => {
+              setActionSheetVisible(true);
+            }}
+            bottomDivider
+            checkmark={
```
Comment (mongomoe) [inline tutorial/rn/components/TaskItem.js:74]: Looks like ListItem.Title does not have a checkmark prop in later versions of react-native-elements. See: https://reactnativeelements.com/docs/listitem/

Comment (mongomoe) [reply tutorial/rn/components/TaskItem.js:74]: I believe you have to move this logic to a ListItem.Checkbox. See: https://reactnativeelements.com/docs/listitem/#listitemcheckbox

**Review by mongomoe — CHANGES_REQUESTED** [review]
Thanks again for submitting a PR and contributing. Some UI bugs in this (left comments explaining) + needs an updated package.json file

**Comment by marklchaves** [comment]
Hello @mongomoe,

Thanks so much for the code review. I'll try to work on your suggestions over the next few days or so.

Cheers! :-)

**Comment by nathan-contino-mongo** [comment]
> Hello @mongomoe,
> 
> Thanks so much for the code review. I'll try to work on your suggestions over the next few days or so.
> 
> Cheers! :-)

Hi Mark,

Any progress on this? We'd like to update the docs to follow this convention, but I also want to make sure you get credit. If you can finish this up in the next couple of days we can avoid having to reimplement it ourselves 😄

**Comment by marklchaves** [comment]
Hey @nathan-contino-mongo,

Thanks for checking in. I'll try to make time to work on this over the weekend! Sorry, I couldn't get to it earlier like I said I would.

Have a great weekend :smile:

File: tutorial/rn/components/TaskItem.js (hunk 1 above)
Comment (marklchaves) [reply tutorial/rn/components/TaskItem.js]: Hi @mongomoe,

I pushed my fixes:

1. Fixed the line spacing in the task action sheet.
2. Fixed the missing task item status in the task list.
3. Updated `react-native-elements` version in package.json.

I didn't use `ListItem.Checkbox`. I kept the [original logic](https://github.com/marklchaves/docs-realm/blob/651c40d0f9d5ded42f66713e1733d37a5c5261f5/tutorial/rn/components/TaskItem.js#L75) that displays the check _icon_ or **In Progress** depending on the status of the task.

Give me a yell if I missed anything 😄

File: tutorial/rn/components/ManageTeam.js [hunk 2]
```diff
@@ -10,58 +10,38 @@ export function ManageTeam() {
   const [newTeamMember, setNewTeamMember] = useState(null);
   const [teamMemberList, setTeamMemberList] = useState([]);
 
-  // :code-block-start: get-team
```
Comment (mongomoe) [inline tutorial/rn/components/ManageTeam.js:13]: The "// :code-block-start", "// :state-start:", ":state-uncomment-end:", "// :state-end:", etc are statements used by our bluehawk markup parser and required for this tutorial. Can you add them back to this PR?

See: https://www.npmjs.com/package/bluehawk

File: tutorial/rn/components/TaskItem.js (hunk 1 above)
Comment (mongomoe) [reply tutorial/rn/components/TaskItem.js]: @marklchaves thanks for upgrading the version + fixing the UI issue

**Review by mongomoe — CHANGES_REQUESTED** [review]
Close to LGTM, the UI looks good. But you just need to bring back the bluehawk comments (see: https://github.com/mongodb/docs-realm/pull/1159/files#r686315829). 

Thanks again for your work on this.

**Review by mongomoe — APPROVED** [review]
LGTM, thanks for all your work on this! Merging this now ✅

**Pull request merged**